packages/react-components/react-search/stories/SearchBox/SearchBoxDefault.stories.tsx

```diff
@@ -1,21 +1,48 @@
 import * as React from 'react';
-import { SearchBox, SearchBoxProps } from '@fluentui/react-search';
+import { ArgTypes } from '@storybook/api';
+import { SearchBox } from '@fluentui/react-search';
+import { Field, makeStyles, useId } from '@fluentui/react-components';
+import type { SearchBoxProps } from '@fluentui/react-search';
 
-import { FilterRegular } from '@fluentui/react-icons';
+const useStyles = makeStyles({
+  root: {
+    // Stack the label above the field
+    display: 'flex',
+    flexDirection: 'column',
+    // Prevent the example from taking the full width of the page (optional)
+    maxWidth: '400px',
+  },
+});
 
-// TODO: split into different stories
-export const Default = (props: Partial<SearchBoxProps>) => (
-  <>
-    <SearchBox {...props} contentAfter={<FilterRegular />} size="small" placeholder="small" />
-    <SearchBox {...props} contentAfter={<FilterRegular />} size="medium" placeholder="medium" />
-    <SearchBox {...props} contentAfter={<FilterRegular />} size="large" placeholder="large" />
-    <SearchBox {...props} contentAfter={null} placeholder="no contentAfter" />
-    <SearchBox {...props} contentAfter={<FilterRegular />} disabled placeholder="disabled" />
-    <SearchBox
-      {...props}
-      contentAfter={<FilterRegular tabIndex={0} onClick={() => console.log('clicked')} />}
-      placeholder="contentAfter button"
-    />
-    <SearchBox root={{ onFocus: () => console.log('test') }} placeholder="custom onFocus" />
-  </>
-);
+export const Default = (props: SearchBoxProps) => {
+  const searchBoxId = useId('searchbox');
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      <Field label="Sample SearchBox">
+        <SearchBox id={searchBoxId} {...props} />
+      </Field>
+    </div>
```

Comment:
You can remove the wrapper on this one and add the styles directly

```suggestion
      <Field className={styles.root} label="Sample SearchBox">
        <SearchBox id={searchBoxId} {...props} />
      </Field>
```